app/src/main/java/org/eyeseetea/malariacare/fragments/AVFragment.java

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright (c) 2015.
+ *
+ * This file is part of QIS Surveillance App.
+ *
+ *  QIS Surveillance App is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  QIS Surveillance App is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with QIS Surveillance App.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.eyeseetea.malariacare.fragments;
+
+import android.app.Activity;
+import android.app.Fragment;
+import android.os.Bundle;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import org.eyeseetea.malariacare.R;
+import org.eyeseetea.malariacare.data.database.model.MediaDB;
+import org.eyeseetea.malariacare.data.repositories.MediaRepository;
+import org.eyeseetea.malariacare.domain.boundary.executors.IAsyncExecutor;
+import org.eyeseetea.malariacare.domain.boundary.executors.IMainExecutor;
+import org.eyeseetea.malariacare.domain.entity.Media;
+import org.eyeseetea.malariacare.domain.usecase.GetMediaUseCase;
+import org.eyeseetea.malariacare.layout.adapters.dashboard.AVAdapter;
+import org.eyeseetea.malariacare.presentation.executors.AsyncExecutor;
+import org.eyeseetea.malariacare.presentation.executors.UIThreadExecutor;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * A placeholder fragment containing a simple view.
+ */
+public class AVFragment extends Fragment implements IDashboardFragment {
```

Comment:
we don´t need to implement IDashboardFragment because this interface was created to work with survey service and broadcast receiver